mo_gymnasium/envs/fruit_tree/fruit_tree.py

```diff
@@ -288,9 +296,35 @@ def __init__(self, depth=6):
         self.current_state = np.array([0, 0], dtype=np.int32)
         self.terminal = False
 
+        # pygame
+        self.row_height = 20
```

Comment:
I guess this should be executed only when `render_mode == "human" or render_mode == "rgb_array`?

Reply:
As far as I checked, other environments like four_room or minecart have similar which executes independently of rener_mode and they call self.render() the mode is human/rgb_array. I would leave it like this for consistency other envs, but can also move it inside of the if statement if you think it's cleaner